src/pages/membership/components/createYouthProfile/CreateYouthProfile.tsx

```diff
@@ -41,6 +44,10 @@ function CreateYouthProflle({ tunnistamoUser }: Props) {
     refetchQueries: ['HasYouthProfile'],
   });
 
+  const toggleErrorNotification = () => {
```

Comment:
I think it would be more readable to call `setShowNotification()` with the correct argument instead of toggling. I would remove this function and call `setShowNotification()` with true or false based on what's supposed to happen.